src/models/hooks/programPersonnel.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable import/prefer-default-export */
+import { Op } from 'sequelize';
+
+const autoPopulateMapsTo = async (sequelize, instance, options) => {
+  console.log('\n\n\n------- Called hook: ', instance.active);
+  if (instance.active === true) {
+    // Update all programPersonnel with the same grantId, programId, and role to map to this id.
+    console.log('\n\n\n------Before', instance.id);
+    let returnValues;
+    try {
+      returnValues = await sequelize.models.ProgramPersonnel.update(
+        { mapsTo: instance.id },
```

Comment:
set active to false

Reply:
Should already be but we can do it here also.